server/utils/azureAppInsights.ts

```diff
@@ -24,7 +24,7 @@ function initialiseAppInsights(): void {
   }
 }
 
-function buildAppInsightsClient(name = defaultName()): TelemetryClient {
+function buildAppInsightsClient(name = defaultName()): TelemetryClient | null {
```

Comment:
These feels similar to `Specify that getToken can return null` - maybe they can go together?